state-sync/state-sync-v2/state-sync-driver/src/utils.rs

```diff
@@ -269,18 +269,9 @@ pub fn fetch_latest_synced_ledger_info(
 
 /// Fetches the latest synced version from the specified storage
 pub fn fetch_latest_synced_version(storage: Arc<dyn DbReader>) -> Result<Version, Error> {
-    let latest_transaction_info =
-        storage
-            .get_latest_transaction_info_option()
-            .map_err(|error| {
-                Error::StorageError(format!(
-                    "Failed to get the latest transaction info from storage: {:?}",
-                    error
-                ))
-            })?;
-    latest_transaction_info
-        .ok_or_else(|| Error::StorageError("Latest transaction info is missing!".into()))
-        .map(|(latest_synced_version, _)| latest_synced_version)
+    storage.get_latest_version().map_err(|e| {
```

Comment:
hmm, I think this is intended to fetch the latest txn info instead. cc: @JoshLind

Reply:
As long as this returns the latest version from storage (i.e., the very latest version even if there has not been a ledger info written to storage for that version), I think it should be okay 😄